var/spack/repos/builtin/packages/zoltan/package.py

```diff
@@ -79,6 +79,8 @@ def install(self, spec, prefix):
                                .format(spec['metis'].prefix.include))
             config_args.append('--with-ldflags=-L{0}'
                                .format(spec['metis'].prefix.lib))
+            if '+int64' in spec['metis']:
```

Comment:
can you handle the -int64 case also explicitly?

Reply:
Sure. I just pushed a commit to handle `~int64`.